packages/directlinespeech/__tests__/utilities/fetchSpeechCredentialsWithCache.js

```diff
@@ -2,15 +2,28 @@ import fetch from 'node-fetch';
 
 const TOKEN_URL_TEMPLATE = 'https://{region}.api.cognitive.microsoft.com/sts/v1.0/issueToken';
 
-async function fromWaterBottle() {
+async function fromWaterBottle(enableInternalHttpSupport = false) {
   const res = await fetch('https://webchat-waterbottle.azurewebsites.net/token/speechservices');
 
   if (!res.ok) {
     throw new Error(`Failed to fetch Cognitive Services Speech Services credentials, server returned ${res.status}`);
   }
 
+  let directLineToken;
```

Comment:
I don't think you need to declare `directLineToken` here since it is only used in the if block.
```suggestion
```